woocommerce/woocommerce-ios
# [Coupons in Orders]Remove addCouponToOrder feature flag.

Closes: #10001 


## Description

With this PR we remove the feature flag hiding the coupons in order feature so it can be released smoothly.

## Testing instructions

In a release build please make sure that the feature is enabled. You can do that by going to the orders tab, tapping to create a new order, and verifying that the + Add Coupon row is there.

## Screenshots

<img src="https://github.com/woocommerce/woocommerce-ios/assets/1864060/0c5713b4-0dbb-429f-aee0-3e6cc27528fd" width="375">

---
